types/node/test/buffer.ts

```diff
@@ -41,7 +41,7 @@ const result2 = Buffer.concat([utf8Buffer, base64Buffer], 9999999);
 // Class Method: Buffer.from(data)
 {
     // Array
-    const buf1: Buffer = Buffer.from([0x62, 0x75, 0x66, 0x66, 0x65, 0x72]);
+    const buf1: Buffer = Buffer.from([0x62, 0x75, 0x66, 0x66, 0x65, 0x72] as ReadonlyArray<number>);
```

Comment:
You can use `as const` to do this:
```suggestion
    const buf1: Buffer = Buffer.from([0x62, 0x75, 0x66, 0x66, 0x65, 0x72] as const);
```

Reply:
:+1: Absolutely, `as const` is the right thing to do here, `as ReadonlyArray<...>` is just so the tests pass on TypeScript 3.2.

Reply:
... however where `typesVersions >= 3.4` I should do as you suggest, thanks! Done :heavy_check_mark: